client/.eslintrc

```diff
@@ -40,7 +40,7 @@ rules:
   react/no-did-update-set-state: 0
   react/no-multi-comp: 2
   react/no-unknown-property: 2
-  react/prop-types: 1
+  react/prop-types: 0
```

Comment:
This is a really useful validation. There's some regression in eslint that's failing this.

At the min, we need to doc why we're doing this change temporarily.